tests/integration/integration_test.py

```diff
@@ -41,6 +41,22 @@ def test_get_query_args(swagger_client):
     assert response.headers['X-Expires-After'] == 'Expiration date'
 
 
+def test_param_multi(swagger_client):
+    result, response = swagger_client.pet.getPetsByIds(
+        petIds=[23, 42],
+    ).result(timeout=1)
+
+    assert len(result) == 2
+    assert result[0]._as_dict() == {
+        'id': 23,
+        'name': 'Takamoto',
+        'photoUrls': [],
+        'category': None,
+        'status': None,
+        'tags': None,
+    }
```

Comment:
Can you add an assertion for result[1] as well please?

Reply:
Sure